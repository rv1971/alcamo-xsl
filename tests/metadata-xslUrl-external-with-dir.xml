<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet type="text/xml" href="../tests/metadata-xslUrl.xsl"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"/>
