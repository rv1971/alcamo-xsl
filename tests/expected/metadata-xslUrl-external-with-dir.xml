<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <a:item>../tests/metadata-xslUrl.xsl</a:item>
  <a:item>../tests/</a:item>
</a:result>
